config/test.exs

```diff
@@ -2,3 +2,11 @@ import Config
 
 config :logger, backends: [{LoggerFileBackend, :log_file}]
 config :logger, :log_file, level: :debug, path: "logs/test.log"
+
+config :task_bunny,
+  disable_auto_start: false,
+  hosts: [
+    default: [
+      connect_options: "amqp://rabbitmq:rabbitmq@localhost:5672/portal?heartbeat=30"
```

Comment:
Should this connect to the portal or default host?

Reply:
Any :)

Reply:
This is my test output, it connects to either host, but my RMQ is configured to accept both guest and rabbitmq users:
![RMQ_VHosts](https://github.com/tsg-global/task-bunny/assets/1404772/3ebc59ad-fa8a-4acc-9e9b-fb2ec9c0129e)



Test output - still a lot of tests failing:
[task_bunny_test_output.txt](https://github.com/tsg-global/task-bunny/files/14297151/task_bunny_test_output.txt)